pages/authentication_page/login.css

```diff
@@ -0,0 +1,95 @@
+.main-nav{
+    display: flex;
+    gap: 0.5rem;
+    padding: 0.5rem 1.5rem;
+    max-width: 100%;
+    height: 76px;
+}
+.main-login{
+    background-color: var(--st-light-pink);
+    max-width: 100%;
+    height: calc(100vh - 80px);
+}
+.brand-name{
+    width: 8rem;
+    color: var(--st-primary);
+}
+.nav-category{
+    display: flex;
```

Comment:
overall code looks very clean and crisp, just one suggestion you can use util class for this

Reply:
yeah i will use that later on ...did this due to time crunch